n64img/image.py

```diff
@@ -76,6 +76,37 @@ def size(self) -> int:
 class CI8(Image):
     pass
 
+# I1, a very primitive type where each bit represents one pixel.
+# Generally, only used for debug fonts and rendered using the cpu instead of the rdp.
+class I1(Image):
+    def __init__(self, data, width, height):
+        super().__init__(data, width, height)
+        self.greyscale = True
+        self.alpha = True
+
+    def parse(self) -> bytes:
+        img = bytearray()
+
+        for x, y, i in iter.iter_image_indexes(
+            self.width, self.height, 0.125, 1, self.flip_h, self.flip_v
+        ):
+
+            b = self.data[i]
+
+            # iterate over bits
+            for j in range(8, 0, -1):
+                # Store the value of each bit as a pixel.
+                p = (b >> (j - 1)) & 0x1
+
+                # Convert active bits to RGB white and inactive to black.
+                p = ceil(0xFF * p)
+
+                img += bytes((p, 0xFF))
```

Comment:
and then I think we don't need the alpha value here at all and we can just append p